### PR TITLE
Improve build

### PR DESCRIPTION
**Pull request by mickaelandrieu**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,13 +38,14 @@ env:
     fast_finish: true        
 
 before_script:
-    - ./tools/move_module.sh
-    - $MODULE_SCRIPTS_DIR/get_prestashop.sh
-    - $MODULE_SCRIPTS_DIR/install_webserver.sh
-    - $MODULE_SCRIPTS_DIR/start_prestashop.sh
-    - $MODULE_SCRIPTS_DIR/install_module.sh gamification
-    
+    - git clone https://github.com/mickaelandrieu/presto-companion.git /tmp/companion
```
Comment (PierreRambaud) [inline .travis.yml]: should it be https://github.com/PrestaShop/module-companion instead?

Comment (mickaelandrieu) [reply .travis.yml]: oh yeah, you're right !

File: .travis.yml [hunk 2]
```diff
@@ -38,13 +38,14 @@ env:
     fast_finish: true        
 
 before_script:
-    - ./tools/move_module.sh
-    - $MODULE_SCRIPTS_DIR/get_prestashop.sh
-    - $MODULE_SCRIPTS_DIR/install_webserver.sh
-    - $MODULE_SCRIPTS_DIR/start_prestashop.sh
-    - $MODULE_SCRIPTS_DIR/install_module.sh gamification
-    
+    - git clone https://github.com/PrestaShop/module-companion.git /tmp/companion
```
Comment (PierreRambaud) [inline .travis.yml]: $COMPANION_DIR instead of /tmp/companion.
And you can use`${HOME}/companion` instead of tmp directory.

Comment (mickaelandrieu) [reply .travis.yml]: not "${HOME} (or you misunderstood what is done here :p)

**Comment by mickaelandrieu** [comment]
Comment addressed => ping @Quetzacoalt91 would you mind to review?

File: .github/main.workflow [hunk 3]
```diff
@@ -0,0 +1,12 @@
+workflow "PrestaShop Module Code Quality" {
+  on = "push"
+  resolves = [
+    "PHP-CS-Fixer"
+  ]
+}
+
+action "PHP-CS-Fixer" {
+  uses = "docker://oskarstark/php-cs-fixer-ga"
+  secrets = ["GITHUB_TOKEN"]
```
Comment (Quetzacoalt91) [inline .github/main.workflow]: Why is the github token provided when we deal with a public repository and no changes applied to the PR?

Comment (mickaelandrieu) [reply .github/main.workflow]: you're right, I've removed it.

**Comment by mickaelandrieu** [comment]
ping @Quetzacoalt91 build is green, i will try to add a Github action in a future pull request :+1:

**Pull request closed without merging**